projects/demo/src/modules/components/pdf-viewer/examples/1/index.ts

```diff
@@ -12,15 +13,25 @@ import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
     encapsulation,
 })
 export class TuiPdfViewerExample1 {
+    private readonly pdf = `assets/media/taiga.pdf`;
+
     constructor(
         @Inject(DomSanitizer) private readonly sanitizer: DomSanitizer,
         @Inject(TuiPdfViewerService) private readonly pdfService: TuiPdfViewerService,
+        @Inject(TUI_IS_MOBILE) private readonly isMobile: boolean,
     ) {}
 
     show(actions: PolymorpheusContent<TuiPdfViewerOptions>): void {
         this.pdfService
             .open(
-                this.sanitizer.bypassSecurityTrustResourceUrl(`/assets/media/taiga.pdf`),
+                this.sanitizer.bypassSecurityTrustResourceUrl(
+                    this.isMobile
+                        ? // Embedded PDFs in mobile doesn't work,
```

Comment:
Can we move this comment above entire ternary? It would be easier to read :)

Reply:
fixed